src/docs/Components/RatingStars/code.mdx

```diff
@@ -159,12 +203,35 @@ These modifiers are to be applied **to the main container** :
 
 <Preview path="stars-result-all-sizes" />
 
+#### With a label
+
+You have the possibility (you are able) to display the average score or the total review count.
+
+To do so, you just need to add a `span.mc-stars-result__label` element in your HTML structure, just after the `span.mc-stars-result__visual` element.
+
+<Preview path="stars-result-label" nude />
+
+#### With a label link
+
+If you want your label element to be a link, you can use an `a` tag instead of a `span` tag, while assigning it the classes specific to the [Link component](/Components/Links/code/#available-sizes).
+
+<Preview path="stars-result-label-link" nude />
+
+#### With an overall score
+
+In case you need to display the global score, you can add the `mc-stars-result__label--score` on the `mc-stars-result__label` element
+
+<Preview path="stars-result-overall" nude />
```

Comment:
@tiloyi Can you change the margin between the stars and the label to 8px (instead of 10px)?

Reply:
fixed